### PR TITLE
enable sphinx_issues as sphinx extension to point to PRs and contribu…

### PR DESCRIPTION
**Pull request by agramfort**
…tor in what's new page

### CODE REVIEW

**Comment by agramfort** [comment]
looks like this:

Screenshot 2022-07-14 at 16 19 39

**Comment by sylvchev** [comment]
Nice ! Thanks @agramfort

File: doc/whatsnew.rst [hunk 1]
```diff
@@ -10,67 +10,69 @@ A catalog of new features, improvements, and bug-fixes in each release.
 v0.2.8.dev
 ----------
 
-- Correct spectral estimation in :func:`pyriemann.utils.covariance.cross_spectrum` to obtain equivalence with SciPy
+- Correct spectral estimation in :func:`pyriemann.utils.covariance.cross_spectrum` to obtain equivalence with SciPy. :pr:`133` by :user:`qbarthelemy`
```
Comment (agramfort) [inline doc/whatsnew.rst:13]: @sylvchev @qbarthelemy @plcrodrigues @gabelstein can you check if I did not screw up the credit assignment of each entry below? 🙏

Comment (sylvchev) [reply doc/whatsnew.rst:13]: ok !

**Review by sylvchev — APPROVED** [review]
Simor minor suggestions on the PR authors.

File: doc/whatsnew.rst (hunk 1 above)
Comment (gabelstein) [reply doc/whatsnew.rst:13]: looks good

File: doc/whatsnew.rst [hunk 2]
```diff
@@ -10,67 +10,69 @@ A catalog of new features, improvements, and bug-fixes in each release.
...
+- Add :func:`pyriemann.utils.distance.distance_harmonic`, and sort functions by their names in code, doc and tests. :pr:`183` by :user:`qbarthelemy`
 
-- Parallelize functions for dataset generation: :func:`pyriemann.datasets.make_gaussian_blobs`
+- Parallelize functions for dataset generation: :func:`pyriemann.datasets.make_gaussian_blobs`. :pr:`179` by :user:`sylvchev`
 
-- Fix dispersion when generating datasets: :func:`pyriemann.datasets.sample_gaussian_spd`
+- Fix dispersion when generating datasets: :func:`pyriemann.datasets.sample_gaussian_spd`. :pr:`179` by :user:`sylvchev`
 
-- Enhance utils functions, to process ndarrays of SPD matrices
+- Enhance base and distance functions, to process ndarrays of SPD matrices. :pr:`187` by :user:`qbarthelemy`
```
Comment (qbarthelemy) [inline doc/whatsnew.rst:71]:
```suggestion
- Enhance base and distance functions, to process ndarrays of SPD matrices. :pr:`186` and :pr:`187` by :user:`qbarthelemy`
```

File: doc/whatsnew.rst (hunk 1 above)
Comment (qbarthelemy) [reply doc/whatsnew.rst:13]: It's ok for me!

**Review by qbarthelemy — APPROVED** [review]

**Pull request merged**